src/uct/ib/mlx5/gdaki/gdaki.c

```diff
@@ -200,8 +200,8 @@ static UCS_CLASS_INIT_FUNC(uct_rc_gdaki_ep_t, const uct_ep_params_t *params)
 err_mem:
     doca_gpu_mem_free(iface->gpu_dev, self->ep_gpu);
 err_ctx:
-    (void)UCT_CUDADRV_FUNC_LOG_ERR(cuCtxPopCurrent(NULL));
-    (void)UCT_CUDADRV_FUNC_LOG_ERR(cuDevicePrimaryCtxRelease(iface->cuda_dev));
+    (void)UCT_CUDADRV_FUNC_LOG_WARN(cuCtxPopCurrent(NULL));
+    (void)UCT_CUDADRV_FUNC_LOG_WARN(cuDevicePrimaryCtxRelease(iface->cuda_dev));
```

Comment:
Missing `err_ctx_release` label here. It should be used at line 69. Otherwise, the context will be popped if push fails, and nothing is pushed.